ignite/contrib/metrics/cohen_kappa.py

```diff
@@ -0,0 +1,63 @@
+from typing import Callable
+
+import torch
+
+from ignite.metrics import EpochMetric
+
+
+class CohenKappa(EpochMetric):
+    """Compute different types of Cohen's Kappa: Non-Wieghted, Linear, Quadratic.
+    Accumulating predictions and the ground-truth during an epoch and applying
+    `sklearn.metrics.cohen_kappa_score <https://scikit-learn.org/stable/modules/
+    generated/sklearn.metrics.cohen_kappa_score.html>`_ .
+
+    Args:
+        output_transform (callable, optional): a callable that is used to transform the
+            :class:`~ignite.engine.engine.Engine`'s ``process_function``'s output into the
+            form expected by the metric. This can be useful if, for example, you have a multi-output model and
+            you want to compute the metric with respect to one of the outputs.
+        weights (str): a string is used to define the type of Cohen's Kappa whether Non-Weighted or Linear or Quadratic
+            (default: "")
+        check_compute_fn (bool): Default False. If True, `cohen_kappa_score
+            <https://scikit-learn.org/stable/modules/generated/sklearn.metrics.cohen_kappa_score.html>`_
+            is run on the first batch of data to ensure there are
+            no issues. User will be warned in case there are any issues computing the function.
+
+    .. code-block:: python
+
+        def activated_output_transform(output):
+            y_pred, y = output
+            return y_pred, y
+
+        weights = None or linear or quadratic
+
+        cohen_kappa = CohenKappa(activated_output_transform, weights)
+
+    """
+
+    def __init__(self, output_transform: Callable = lambda x: x, weights: str = "", check_compute_fn: bool = False):
+        # initalize weights
+        if weights == "":
+            self.weights = None
+        else:
+            self.weights = weights
```

Comment:
```suggestion
        self.weights = weights
```

Reply:
This doesn't pass the CI, because of conversion between `NoneType`, `str`. The weights takes a `None` or `linear` of `quadratic`. So i made this hack to pass the CI